deployment/load-nci-dump.bash

```diff
@@ -1,14 +1,22 @@
 #!/usr/bin/env bash
 
 set -eu
+set -x
 umask 0022
 
 # NCI dates are in AEST
 export TZ="Australia/Sydney"
 
+# Remove any previous pgdump files before processing cubedash job
+rm -rf /data/nci/*-datacube.pgdump
+
 # Optional first argument is day to load (eg. "yesterday")
 dump_id="$(date "-d${1:-today}" +%Y%m%d)"
-psql_args="-h db-prod-eks-datacube-default.cfeq4wxgcaui.ap-southeast-2.rds.amazonaws.com -U superuser"
+export PGUSER="superuser"
+export PGHOST="db-prod-eks-datacube-default.cfeq4wxgcaui.ap-southeast-2.rds.amazonaws.com"
+export PGPORT=5432
```

Comment:
These should be pulled from env vars, not hard coded as env vars

Reply:
Updated